dbt_project.yml

```diff
@@ -114,3 +137,12 @@ models:
       +schema: 'scratch'
       +tags: 'snowplow_media_player_incremental'
       +enabled: false
+    media_ad_views:
```

Comment:
I tried executing the package without enabling the relevant variable and it still generated empty tables for me. Can we put an `enable` config to be dependent on the variable being true for all the optional modules (I think it's media_ads / media_ad_views)?

Reply:
Good point! I have conditionally disabled the ad views and ads models (in the model config header) based on whether the ad context is enabled.